preview_generator/preview/builder/image__imconvert.py

```diff
@@ -16,35 +15,36 @@
 
 from preview_generator.exception import BuilderDependencyNotFound
 from preview_generator.exception import IntermediateFileBuildingFailed
+from preview_generator.extension import mimetypes_storage
 from preview_generator.preview.builder.image__pillow import ImagePreviewBuilderPillow  # nopep8
 from preview_generator.preview.generic_preview import ImagePreviewBuilder
 from preview_generator.utils import ImgDims
+from preview_generator.utils import MimetypeMapping
 from preview_generator.utils import executable_is_available
 
 
 class ImagePreviewBuilderIMConvert(ImagePreviewBuilder):
 
     MIMETYPES = []  # type: typing.List[str]
-    SUPPORTED_RAW_CAMERA_MIMETYPE = [
-        "image/x-sony-arw",
-        "image/x-adobe-dng",
-        "image/x-sony-sr2",
-        "image/x-sony-srf",
-        "image/x-sigma-x3f",
-        "image/x-canon-crw",
-        "image/x-canon-cr2",
-        "image/x-epson-erf",
-        "image/x-fuji-raf",
-        "image/x-nikon-nef",
-        "image/x-olympus-orf",
-        "image/x-panasonic-raw",
-        "image/x-panasonic-rw2",
-        "image/x-pentax-pef",
-        "image/x-kodak-dcr",
-        "image/x-kodak-k25",
-        "image/x-kodak-kdc",
-        "image/x-minolta-mrw",
-        "image/x-kde-raw",
+    SUPPORTED_RAW_CAMERA_MIMETYPE_MAPPING = [
```

Comment:
why do you use a list of lists instead of a dictionnary which would be more clear ?

By the way, as soon as you introduce mapping, you should consider potentially several extensions for a given mime type. Example `image/jpeg` may be `.jpeg` or `.jpg`

So I would do sometihng like this:
```
SUPPORTED_RAW_CAMERA_MIMETYPE_MAPPING = {
         "image/x-sony-arw": [".arw"],
}
```

Reply:
But... you also defined below a class `MimetypeMapping`, so why not using your class ?

```
SUPPORTED_RAW_CAMERA_MIMETYPE_MAPPING = [
         MimetypeMapping("image/x-sony-arw", [".arw"]),
         MimetypeMapping(""image/x-adobe-dng", [".dng"]),
]
```

and implement a function `mapping_list_as_dict(SUPPORTED_RAW_CAMERA_MIMETYPE_MAPPING) -> dict` ?

Reply:
Because dictionary may cause issue in some specific case. 
There is some case where one extension can mean multiple mimetype like ".ogg", so i choose to use list to support all cases.

Reply:
By the way, your design is not very generic, I think. For example, getting mimetype from extension and reverse operation are not done using the same data (or I missed something)

Reply:
added todo, see #159 for refactoring here